src/sentry/tasks/post_process.py

```diff
@@ -1180,27 +1179,15 @@ def sdk_crash_monitoring(job: PostProcessJob):
     if not features.has("organizations:sdk-crash-detection", event.project.organization):
         return
 
-    cocoa_project_id = options.get(
-        "issues.sdk_crash_detection.cocoa.project_id",
-    )
-    if not cocoa_project_id or cocoa_project_id == 0:
-        sentry_sdk.capture_message("Cocoa project_id is not set.")
+    configs = build_sdk_crash_detection_configs()
+    if not configs or len(configs) == 0:
```

Comment:
An empty list is a falsy value, so this has the same effect:
```suggestion
    if not configs:
```

Reply:
TIL; Will change that in the next PR.